CMakeLists.txt

```diff
@@ -107,6 +107,11 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE RELEASE)
 endif()
 
+# Build with link time optimizations in release mode by default. Brings noticeable speedup.
+if(CMAKE_BUILD_TYPE STREQUAL "RELEASE")
+  set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
+endif()
```

Comment:
shouldn't this be gated by the BUILD_GUI as well?

Reply:
Yes, I hadn't gotten around to it then. It is gated now.

Reply:
If we don't use it with BUILD_GUI then it will hardly ever be used as almost everyone builds with the GUI.   We need to determine if it works or not with the GUI enabled.

Reply:
Its unclear to me if that means you just don't get LTO on Qt without building from source or if you can't link such a library at all when using LTO.  I suspect it is the former as people don't rebuild libc from scratch to use lto.